extensions/postinstall.mjs

```diff
@@ -26,7 +26,6 @@ function processRoot() {
 function processLib() {
 	const toDelete = new Set([
 		'tsc.js',
-		'tsserverlibrary.js',
```

Comment:
I'm curious what this affects; is this then included in the extension? Or is this deleting it from node modules such that it's not there at all?

Reply:
It makes no difference in dist/browser, but (1) I don't have a non-browser build to check (2) it is still in extensions/node_modules/typescript/lib, which might ship with desktop vscode. I don't know.

Reply:
How do I make that happen? 

(And ideally the web would not ship tsserver, just tsserver.web.js, but I expect that's already what happens.)

Reply:
I'm sorta confused as to why this script modified node_modules at all; that isn't something that would be shipped to end users, right? Is there another step somewhere that is copying the files instead? That seems much clearer to me.

Reply:
> Is there another step somewhere that is copying the files instead?

That is the key question I think.

Reply:
What we can try is moving this list to a `extensions/.vscodeignore` file. The current approach ensures the dev TS code matches the TS code we'd have in prod, which is nice for confidence in what we are shipping. I guess the other option is to install another full copy of TS as a dev dependency so we can pull in this one file

Reply:
I added to .vscodeignore. It was pretty simple, because the code in postinstall's processLib is way more complicated than the contents of typescript/lib justify; only the 3 files in `toDelete` really matter.